### DISCUSSION TITLE
Does Netbox have an EULA?

### DISCUSSION THREAD

**Discussion by K3Mi** [netbox-community/netbox#9195] [Q&A] [answered]
@jeremystretch - we are looking to test out netbox in our environment. Our environment is becoming increasingly locked down and any new software requires review from Security, software compliance, and legal teams.  Is there any information available regarding a EULA or is there none at the moment for netbox?

**Comment by jcollie** [comment, marked as answer]
Netbox is open source under the terms of the Apache License 2.0. EULAs are typically only used on closed, proprietary software.

https://github.com/netbox-community/netbox/blob/develop/LICENSE.txt

**Reply by K3Mi** [reply]
Thanks @jcollie
